src/tls/mod.rs

```diff
@@ -19,26 +16,27 @@ mod settings;
 #[cfg(feature = "sources-tls")]
 pub(crate) use incoming::{MaybeTlsIncomingStream, MaybeTlsListener};
 pub(crate) use maybe_tls::MaybeTls;
-pub(crate) use outgoing::MaybeTlsConnector;
 pub use settings::{MaybeTlsSettings, TlsConfig, TlsOptions, TlsSettings};
 
+// TODO: Rename to TlsResult?
 pub type Result<T> = std::result::Result<T, TlsError>;
 
 pub type MaybeTlsStream<S> = MaybeTls<S, SslStream<S>>;
 
+// TODO: check what not used and remove
 #[derive(Debug, Snafu)]
 pub enum TlsError {
```

Comment:
Can I rename this enum to `Error`? Then import can be used as `crate::tls::Result` / `crate::tls::Error` (same as in root: `crate::Result` / `crate::Error`).

Reply:
I don't see that as much of a benefit. I would tend to go the other way, to rename `Result` to `TlsResult` so we don't have to use the module path on each use. Looking at our sources, we appear to have far more instances of `pub (enum|type|struct) SomethingError` and `SomethingResult` than the other. In fact, the above is the only `type Result` other than the crate level type alias.